packages/arb-token-bridge-ui/src/state/app/state.ts

```diff
@@ -123,7 +126,13 @@ export type AppState = {
   pwLoadedState: PendingWithdrawalsLoadedState
   arbTokenBridgeLoaded: boolean
 
-  changeNetwork: ((network: L1Network | L2Network) => Promise<void>) | null
+  changeNetwork:
+    | (({
+        chainId,
+        onSuccess,
+        onError
+      }: ChangeNetworkProps) => Promise<void>)
+    | null
```

Comment:
Let's get rid of `changeNetwork` from the app state completely, and just call the util function.